package/MDAnalysis/core/topologyattrs.py

```diff
@@ -518,6 +518,15 @@ def set_segments(self, sg, values):
         """Set segmentattributes for a given SegmentGroup"""
         raise NotImplementedError
 
+    @classmethod
+    def are_values_missing(self, values):
+        """check if an attribute has a missing value
+        .. versionadded:: 2.7.0
```

Comment:
```suggestion

        .. versionadded:: 2.7.0
```

Needs a newline I think